SPECS/pytorch/pytorch.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
     "Signatures": {
-        "pytorch-2.0.0-submodules.tar.gz": "81fa75ec16c2228fc75ae8e0c7e5e841665c50581995f48994a490c9a2a87af4",
-        "pytorch-2.0.0.tar.gz": "cecc38b6d4256b810336edfc6119d7a57b701fdf1ba43c50001f31e2724fd8e2"
+        "pytorch-2.2.2-submodules.tar.gz": "74d91f9cbba81848a0c07c718810889c46ca2d24a198444d8e3caca13eea9ffc",
```

Comment:
Test build seems to be broken

Reply:
falak, it's not broken. Actually, it has an unresolved dependency due to runtime dependency on sympy which has a runtime dependency on xorg-x11-server, which was failing.  pytorch -> sympy -> xorg-x11-server.
I recently had a [PR to fix this build failure for xorg-x11-server.](https://github.com/microsoft/azurelinux/pull/8657)

Reply:
once yesterday's dailybuild for 3.0-dev is done, this same buddybuild should not cause any failure.